bitsv/network/services/bchsvexplorer.py

```diff
@@ -86,7 +87,7 @@ def get_unspents(cls, address):
         r = requests.get(cls.MAIN_UNSPENT_API.format(address), timeout=DEFAULT_TIMEOUT)
         r.raise_for_status()  # pragma: no cover
         utxos = [
-            Unspent(amount=currency_to_satoshi(utxo['amount'], 'bsv'),
+            Unspent(amount=int(utxo['value']),
```

Comment:
I will revert this because wierdly their api mixes and matches bsv units here and satoshi units for `get_balance`... but for bitsv we normalize to satoshis for everything coming from 3rd party APIs